dockstore-webservice/src/main/java/io/dockstore/webservice/core/AppTool.java

```diff
@@ -54,12 +55,12 @@ public EntryTypeMetadata getEntryTypeMetadata() {
     }
 
     @Override
-    public Entry getParentEntry() {
+    public Set<Entry> getParentEntry() {
         return null;
```

Comment:
Your code works, but I'm still wondering if this shouldn't return an empty set instead?

Reply:
This is the comment above about sonarcloud, when I tried that I got 
```
ERROR] testChangingAppToolTopics  Time elapsed: 3.923 s  <<< ERROR!
jakarta.ws.rs.ProcessingException: Error reading entity from input stream.
	at io.dockstore.webservice.SwaggerWebhookIT.testChangingAppToolTopics(SwaggerWebhookIT.java:1279)
Caused by: com.fasterxml.jackson.databind.exc.MismatchedInputException: 
Cannot deserialize value of type `io.swagger.client.model.Entry` from Array value (token `JsonToken.START_ARRAY`)
 at [Source: (org.glassfish.jersey.message.internal.ReaderInterceptorExecutor$UnCloseableInputStream); line: 1, column: 1059] (through reference chain: io.swagger.client.model.AppTool["parentEntry"])
	at io.dockstore.webservice.SwaggerWebhookIT.testChangingAppToolTopics(SwaggerWebhookIT.java:1279)
```

Looking more closely at the message, maybe this would instantly break older swagger clients?

Reply:
One option could be to return an arbitrary one in the old property and return an array in a new property